### ISSUE TITLE
Issues found using type-checker

### ISSUE THREAD

**Issue by just-boris** [ai/size-limit#116]
While I was performing the migration to Typescript (https://github.com/ai/size-limit/pull/115). I found a few actual issues, that may be fixed regardless.

**1. Documentation issue. Entry can be just a string, not only array**

https://github.com/ai/size-limit/blob/master/index.js#L200

**2. When `size-limit` cannot find a package.json, it crashes**

Steps to reproduce:
1. Create `.size-limit.js` in a random folder without package.json
2. Run `npx size-limit`

You will see a confusing exception:

```
ERROR  TypeError: Cannot read property 'package' of undefined
  at Promise.all.configFile.config.map (<root-dir>/size-limit-test/node_modules/size-limit/cli-body.js:336:38)
  at Array.map (<anonymous>)
  at run (<root-dir>/size-limit-test/node_modules/size-limit/cli-body.js:335:54)
  at <anonymous>
```

Expected: size-limit works fine in this environment. `package.json` presence is not a requirement

**3. Useless `ignore` field**

https://github.com/ai/size-limit/blob/master/cli-body.js#L391

In this object, it does nothing. It should be in the `files` objects, but not at the top level.

**4. Exception when an empty array is provided**

This code

```js
getSize([], {webpack: false}).then(size => {
  console.log(size);
});
``` 

crashes:

```
(node:8842) UnhandledPromiseRejectionWarning: TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at getSize (<root-dir>/size-limit-test/node_modules/size-limit/index.js:233:18)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
    at Function.Module.runMain (module.js:696:11)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
```

Expected: more user-friendly error or all zeros in response

**Comment by ai** [comment]
Thanks. It is very interesting. I will try to investigate them a few days later.

**Comment by ai** [comment]
> 1. Documentation issue. Entry can be just a string, not only array

b5c065d

> 2. When size-limit cannot find a package.json, it crashes

I will fix it later when I will find how to create a test

> 3. Useless ignore field

842465b

> 4. Exception when an empty array is provided

359b236 (I also fixed JSDoc for this case)

**Comment by ai** [comment]
> 2. When size-limit cannot find a package.json, it crashes

ea78335

**Comment by ai** [comment]
Released in 1.3.6

**Comment by just-boris** [comment]
Thank you!
Very happy to make this project a little more stable
